src/include/distributed/foreign_constraint.h

```diff
@@ -0,0 +1,24 @@
+/*-------------------------------------------------------------------------
+ * foreign_constraint.h
+ *
+ * Copyright (c) 2016, Citus Data, Inc.
+ *
+ *-------------------------------------------------------------------------
+ */
+
+#ifndef FOREIGN_CONSTRAINT_H
+#define FOREIGN_CONSTRAINT_H
+
+#include "postgres.h"
+#include "postgres_ext.h"
+#include "utils/relcache.h"
+#include "nodes/primnodes.h"
+
+extern void ErrorIfUnsupportedForeignConstraint(Relation relation, char
+												distributionMethod,
+												Var *distributionColumn, uint32
+												colocationId);
+extern List * GetTableForeignConstraintCommands(Oid relationId);
```

Comment:
Minor: Wrt to the coding conventions, we should probably switch the order of these two functions: `GetTableForeignConstraintCommands ()` and `TableReferenced ()`